python-sdk/src/astro/databases/base.py

```diff
@@ -248,7 +248,7 @@ def create_table_using_schema_autodetection(
         table: BaseTable,
         file: File | None = None,
         dataframe: pd.DataFrame | None = None,
-        columns_names_capitalization: ColumnCapitalization = "lower",  # skipcq
+        columns_names_capitalization: ColumnCapitalization = "original",  # skipcq
```

Comment:
This is a breaking change. Will this be part of 1.2? Are we happy to introduce breaking changes as part of a minor release? Would it make sense to keep the old default, but support original - and encourage users to use the "original" capitalization?

Reply:
@tatiana This was one of the requirements of - https://github.com/astronomer/astro-sdk/issues/917 to change the default to the original. 

cc: @kaxil

Reply:
@tatiana I think we can consider the original functionality as a bug. It makes no sense to modify data by default.

Reply:
The original functionality is confusing for Astro Build users directly working with output data in subsequent cells. Unaware of the SDK internals, users expect to e.g. work in a Python task with columns in the same casing as they were in an upstream SQL task.

We are looking for a way to disable this without an explicit flag in the DAG code. We initially thought an option could be to define casing with an env variable we could set for all projects; but this would cause an inconsistency for Pipelines exported to deployments.

Astro Build aside, it makes sense that the default behavior is to not change casing. Especially if it's not clearly obvious to the user.